support-frontend/assets/helpers/productPrice/__tests__/priceDescriptionsDigitalTests.js

```diff
@@ -62,13 +62,13 @@ describe('getBillingDescription', () => {
   it('should return a description of the price', () => {
 
     expect(getBillingDescription(productPriceMonthlyWithDiscount, Monthly))
-      .toBe('You\'ll pay £5.99/month for 3 months, then £11.99/month');
+      .toBe('You\'ll pay £5.99/month for 3 months, then £11.99 per month');
```

Comment:
Ideally this would be '£5.99 a month' or '£5.99 per month' but I know the code has to support that! :-)